test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -622,5 +624,21 @@ public void Fluent_Clear_Set_Append_WritesExpectedContent()
             // Assert
             Assert.Equal(expected, tagHelperContent.GetContent());
         }
+
+        [Fact]
+        public void WriteTo_WritesToATextWriter()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var writer = new StringWriter();
+            tagHelperContent.SetContent("Hello ");
+            tagHelperContent.Append("World");
+
+            // Act
+            tagHelperContent.WriteTo(writer, new HtmlEncoder());
```

Comment:
Always use the test encoders. Even if it makes no difference. Someone will have to look at this later and go ???